prysm.sh

```diff
@@ -181,7 +181,12 @@ if [[ $1 == beacon-chain ]]; then
     if [[ ! -x $BEACON_CHAIN_REAL ]]; then
         color "34" "Downloading beacon chain@${prysm_version} to ${BEACON_CHAIN_REAL} (${reason})"
         file=beacon-chain-${prysm_version}-${system}-${arch}
-        curl -L "https://prysmaticlabs.com/releases/${file}" -o "$BEACON_CHAIN_REAL"
+        res=$(curl -w '%{http_code}\n' -L "https://prysmaticlabs.com/releases/${file}"  -o "$BEACON_CHAIN_REAL")
+        if [[ $res == 404 ]];then
+          echo "404 error, exit"
```

Comment:
Can you please add a more meaningful message? For example:
```
No prysm beacon chain found for release $prysm_version
```

Reply:
addressed.